### DISCUSSION TITLE
when will a native Apple Silicon (Arm)...

### DISCUSSION THREAD

**Discussion by ithakaa** [rustdesk/rustdesk#3969] [General]
Version be available?

**Comment by WhiteAls** [comment]
see #3795

**Reply by ithakaa** [reply]
what am I seeing ?

**Comment by lollilol** [comment]
does no developer of rustdesk have an apple silicon? Can we somehow support you? would love to see apple silicon support

**Reply by rustdesk** [reply]
we have no apple silicon.
